### ISSUE TITLE
Focus ring not fully visible

### ISSUE THREAD

**Issue by aliniacb** [tailwindlabs/tailwindui-issues#1348]
**What component (if applicable)**
- URL for category:  https://tailwindui.com/components/application-ui/page-examples/home-screens
- Component name: Sidebar component

**Describe the bug**

**To Reproduce**
1. Go to https://tailwindui.com/components/application-ui/page-examples/home-screens first example
2. Click on the sidebar component (user context menu)
3. Once clicked the focus ring appears and the upper part is cut

**Expected behavior**
The focus ring should be fully visible

**Screenshots**

![Screenshot 2022-10-11 at 22 16 47](https://user-images.githubusercontent.com/25739439/195179608-3f6810b7-9c22-49be-8e4f-514d3ce79099.png)

**Browser/Device (if applicable)**
 - ALL

**Additional context**

Spacing the component with padding rather than margin fixes the issue

**Current:**
```html
 <!-- Sidebar component, swap this element with another sidebar if you like -->
    <div class="mt-6 flex h-0 flex-1 flex-col overflow-y-auto">
     ...
    </div>
```
**To fix:**
```html
 <!-- Sidebar component, swap this element with another sidebar if you like -->
    <div class="pt-6 flex h-0 flex-1 flex-col overflow-y-auto">
     ...
    </div>
```

**Comment by bradlc** [comment]
Hey @aliniacb, thanks for reporting, this is now fixed 👍
